models/task/task.go

```diff
@@ -1,6 +1,7 @@
 package task
 
 import (
+	"../../config"
```

Comment:
__[gometalinter]__
* [gotype] could not import ../../config (can't find import: /sideci/fascia/config)